<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>11</y>
      <width>361</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_1">
         <property name="text">
          <string>监听地址</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboIP_5">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>0</height>
          </size>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
         <property name="duplicatesEnabled">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>127.0.0.1</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>监听端口</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinPort_5">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>65535</number>
         </property>
         <property name="value">
          <number>1200</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0" colspan="2">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLineEdit" name="editMsg_5"/>
         </item>
         <item>
          <widget class="QToolButton" name="btnSend_5">
           <property name="text">
            <string>发送消息</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>387</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="act_art"/>
   <addaction name="act_stop"/>
   <addaction name="act_clear"/>
   <addaction name="act_Exit"/>
  </widget>
  <action name="act_art">
   <property name="text">
    <string>start</string>
   </property>
  </action>
  <action name="act_stop">
   <property name="text">
    <string>stop</string>
   </property>
  </action>
  <action name="act_clear">
   <property name="text">
    <string>clear</string>
   </property>
  </action>
  <action name="act_Exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
